plugins/eddn.py

```diff
@@ -270,14 +270,22 @@ def export_outfitting(self, data: Mapping[str, Any], is_beta: bool) -> None:
         :param data: dict containing the outfitting data
         :param is_beta: whether or not we're currently in beta mode
         """
-        modules: Dict[str, Any] = data['lastStarport'].get('modules') or {}
+        modules: Dict[str, Any] = data['lastStarport'].get('modules')
+        if not modules:
+            logger.debug('modules was None')
```

Comment:
if you want to check for None, use `is None`, other things resolve to False, such as an empty dict